<!--rosbag_record.launch-->
<launch>

<!-- <arg name="path" default="$(env HOME)/catkin_ws/rosbags/" /> -->
<arg name="path" default="/media/user1/Images/rosbags/" />

<node pkg="rosbag" type="record" name="record_bag1" respawn="true" args="-o $(arg path)pointcloud.bag /velodyne_points"/>
<node pkg="rosbag" type="record" name="record_bag2" respawn="true" args="-o $(arg path)camera.bag /camera/linefiltered /camera/rectified"/>
<node pkg="rosbag" type="record" name="record_bag0" respawn="true" args="-o $(arg path)other.bag -x '/velodyne_points|/camera/(.*)'"/>

</launch>



